packages/database/src/core/util/util.ts

```diff
@@ -179,14 +167,8 @@ export const fatal = function(...var_args: string[]) {
  * @param {...*} var_args
```

Comment:
This comment is for the hidden block just above (line 173): I feel that we should also log 'fatal' messages to the user-specified log functions.

Reply:
Done